### PR TITLE
fix warnings and enhanced mobile view UI

### LINKED ISSUE #16: Enhancement UI rendering in mobile view.
Some of the elements in the mobile view are not within the boundary.

WhatsApp Image 2020-09-20 at 7 40 42 AM

@iiitv/project-maintainers i would like to work on this issue.

### PR DESCRIPTION
**Pull request by nlok5923**
fixes : #16 

before changes : 
WhatsApp Image 2020-09-20 at 7 40 42 AM (1)

After changes :
WhatsApp Image 2020-09-23 at 4 06 38 AM

Deployment link : https://mystifying-hoover-1080d8.netlify.app/

### CODE REVIEW

**Comment by nlok5923** [comment]
@iiitv/project-maintainers sir please review this pull request.

File: src/component/clubs/club-left-part/club-left-part.css [hunk 1]
```diff
@@ -36,3 +36,10 @@
   background-color: #2a9adb;
   color: #fff;
 }
+@media only screen and (max-width : 800px)
+{
+  .club-left-part button {
+    padding: 2px 10px 2px 10px;
+    font-size: 12px;
+  }
+}
```
Comment (aashutoshrathi) [inline src/component/clubs/club-left-part/club-left-part.css]: Make sure you add new lines at the end.

File: src/component/clubs/club-right-part/club-right.css [hunk 2]
```diff
@@ -27,4 +27,19 @@
...
+    .club-name{
+        font-size:13px
+    }
+    .club-right-part h4{
+        font-size : 13px;
+    }
+    .club-right-part p{
+        font-size : 12px 
+    }
 }
```
Comment (aashutoshrathi) [inline src/component/clubs/club-right-part/club-right.css]: Same here.

File: src/component/clubs/club-right-part/club-right.css [hunk 3]
```diff
@@ -27,4 +27,19 @@
...
+        height: 8%;
+    }
+    .club-name{
+        font-size:13px
+    }
+    .club-right-part h4{
+        font-size : 13px;
+    }
+    .club-right-part p{
+        font-size : 12px
```
Comment (aashutoshrathi) [inline src/component/clubs/club-right-part/club-right.css]: Also, try using `em` or `rem` for `font-size`

File: src/main-page/navbar/main navbar/navbar.css [hunk 4]
```diff
@@ -205,6 +205,12 @@ body {
     padding-bottom: 0;
     z-index: 99;
   }
+  .logo h1{
+    font-size :20px
+  }
+  .logo h4{
```
Comment (aashutoshrathi) [inline src/main-page/navbar/main navbar/navbar.css]: I think there are spacing issues here `{`

File: src/main-page/navbar/main navbar/navbar.js [hunk 5]
```diff
@@ -3,7 +3,7 @@ import { NavLink } from "react-router-dom";
 import { connect } from "react-redux";
 import "./navbar.css";
 const Navbar = (props) => {
-  const click = 0;
+  let click = 0;
   const myFunction = () => {
```
Comment (aashutoshrathi) [inline src/main-page/navbar/main navbar/navbar.js:7]: Can we name this component `Navbar`

Comment (aashutoshrathi) [reply src/main-page/navbar/main navbar/navbar.js:7]: Also, I see a space in path can you fix that to use recommended naming strategy.

**Review by aashutoshrathi — CHANGES_REQUESTED** [review]
I left a few comments on code, once you fix this we will move on to preview review.

File: src/main-page/navbar/main navbar/navbar.js (hunk 5 above)
Comment (nlok5923) [reply src/main-page/navbar/main navbar/navbar.js:7]: @aashutoshrathi  sir can you please elaborate it a bit.

Comment (aashutoshrathi) [reply src/main-page/navbar/main navbar/navbar.js:7]: - `src/main-page/navbar/main navbar/navbar.js` the directory here contains space in name.
- Also, the component's name is not correct here, please change that too

**Comment by nlok5923** [comment]
@iiitv/project-maintainers sir i had done changes as said by @aashutoshrathi sir please review this pr.

**Review by theyashshahs — COMMENTED** [review]
@nlok5923 add an ss of work

**Comment by nlok5923** [comment]
@yashshah2820 sir ss : 
WhatsApp Image 2020-09-26 at 10 32 19 AM

File: src/component/clubs/club-left-part/club-left-part.css (hunk 1 above)
Comment (theyashshahs) [inline src/component/clubs/club-left-part/club-left-part.css:45]: add a line at the end

File: src/component/clubs/club-right-part/club-right.css [hunk 6]
```diff
@@ -27,4 +27,19 @@
...
+    .club-name{
+        font-size: 0.8em;
+    }
+    .club-right-part h4{
+        font-size : 1em;
+    }
+    .club-right-part p{
+        font-size : 0.8em; 
+    }
+}
```
Comment (theyashshahs) [inline src/component/clubs/club-right-part/club-right.css:45]: same here

**Review by theyashshahs — CHANGES_REQUESTED** [review]

**Comment by theyashshahs** [comment]
@nlok5923 also check there is a line at the end of every file that you have changed

**Comment by nlok5923** [comment]
@yashshah2820 added new line at the end of each file i changed.

**Review by theyashshahs — APPROVED** [review]

**Comment by nlok5923** [comment]
@iiitv/project-maintainers sir please review this pull request.

**Review by AmanRaj1608 — APPROVED** [review]

**Pull request merged**